front/tests/stdcm-page-model.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/prefer-default-export */
+import { expect, Locator, Page } from '@playwright/test';
+import infraManagement from '../public/locales/fr/infraManagement.json';
+
+export class PlaywrightSTDCMPage {
+  // The current page object
```

Comment:
```suggestion
```
I would remove these comments as named of the variables are clear enough

Reply:
Thanks for the comments. Okay, if it's clear, I'll remove the comments.

Reply:
Done